### PR TITLE
Keep comments for ts interface generation :sparkles:

### PR DESCRIPTION
**Pull request by Freymaurer**
I updated ts interface generation to keep xmlDoc comments from f# code. I tried to uphold existing patterns, but please check if everything is fine with you:

The changes transpile the following f# code:

```fsharp
/// This is a user, a very strange individium which will find each and every edge case existing.
[<AllowNullLiteral>]
[<Global>]
type User [<ParamObjectAttribute; Emit("$0")>] (id: int, ?name: string, ?age: int) =
    /// Id used for id stuff
    member val id = id with get, set
    /// name used for naming conventions
    member val name: string option = jsNative with get, set
    /// age because you never tell age
    member val age: int option = jsNative with get, set
```

to:

```ts
/**
 * This is a user, a very strange individium which will find each and every edge case existing.
 */
export interface User {
    /**
     * Id used for id stuff
     */
    id: int32,
    /**
     * name used for naming conventions
     */
    name?: string,
    /**
     * age because you never tell age
     */
    age?: int32
}
```

I kept the example in `src/quicktest/QuickTest.fs`.

### CODE REVIEW

File: src/quicktest/QuickTest.fs [hunk 1]
```diff
@@ -94,7 +94,23 @@ let measureTime (f: unit -> unit) : unit =
 
 printfn "Running quick tests..."
 
+
+/// This is a user, a very strange individium which will find each and every edge case existing.
```
Comment (ncave) [inline src/quicktest/QuickTest.fs:98]: *individuum

File: src/Fable.Transforms/Fable2Babel.fs [hunk 2]
```diff
@@ -3788,6 +3788,10 @@ module Util =
         (classDecl: Fable.ClassDecl)
         (ent: Fable.Entity)
         =
+        printfn "[classDecl] %A" classDecl
```
Comment (ncave) [inline src/Fable.Transforms/Fable2Babel.fs:3791]: I assume these were debug statements, can they be removed?

**Comment by Freymaurer** [comment]
Is there some contribution guide i can follow or just try to follow the existing patterns?

**Comment by MangelMaxime** [comment]
> Is there some contribution guide i can follow or just try to follow the existing patterns?

Just try to follow the existing convention for this PR it would be something like:

```md
* [TS] Include XML Doc comment on interface properties (by @Freymaurer)
```

File: src/Fable.Transforms/Fable2Babel.fs (hunk 2 above)
Comment (Freymaurer) [reply src/Fable.Transforms/Fable2Babel.fs:3791]: Yes sorry!

**Comment by Freymaurer** [comment]
@ncave done!

File: src/Fable.Transforms/Fable2Babel.fs [hunk 3]
```diff
@@ -3797,6 +3798,12 @@ module Util =
                 |> List.mapi (fun index arg ->
                     let name = defaultArg arg.Name $"arg{index}"
 
+                    /// Try to find getter/setter in f# syntax for POJOs. If found propagate its xml doc to interface.
+                    let tryXmlDoc =
+                        ent.MembersFunctionsAndValues
```
Comment (ncave) [inline src/Fable.Transforms/Fable2Babel.fs:3803]: This lookup may be a bit brittle when there are overloads. I know we don't normally allow overloads on interfaces, unless the mangle attribute is added, so it should be mostly fine, at least as a first pass.

**Review by ncave — APPROVED** [review]
LGTM

**Comment by MangelMaxime** [comment]
Thank you
